app/1.0/docs/devguide/data-binding.md

```diff
@@ -88,412 +123,257 @@ annotation inside the child element.
-    is: 'custom-element',
-    properties: {
-      someProp: {
-        type: String,
-        notify: true
-      }
-    }
-  });
-</script>
-...
+To bind to an attribute, use `$=` rather than `=`:
 
-<!-- changes to "value" propagate downward to "someProp" on child -->
-<!-- changes to "someProp" propagate upward to "value" on host  -->
-<custom-element some-prop="{{value}}"></custom-element>
 ```
-
-Example 2: One-way binding (downward) { .caption }
-
+<div style$="color: {{myColor}};">
 ```
-<script>
-  Polymer({
-    is: 'custom-element',
-    properties: {
-      someProp: {
-        type: String,
-        notify: true
-      }
-    }
-  });
-</script>
 
-...
+Where the attribute value is either a [binding annotation](#binding-annotation) or a [compound
+binding](#compound-binding).
 
-<!-- changes to "value" propagate downward to "someProp" on child -->
-<!-- changes to "someProp" are ignored by host due to square-bracket syntax -->
-<custom-element some-prop="[[value]]"></custom-element>
-```
+Attribute binding results in a call to:
 
-Example 3: One-way binding (downward) { .caption }
 
+```js
+element.setAttribute(attr,value);
 ```
-<script>
 
-  Polymer({
-    is: 'custom-element',
-    properties: {
-      someProp: String    // no notify:true!
-    }
-  });
+As opposed to:
 
-</script>
-...
 
-<!-- changes to "value" propagate downward to "someProp" on child -->
-<!-- changes to "someProp" are not notified to host due to notify:falsey -->
-<custom-element some-prop="{{value}}"></custom-element>
+```js
+element.property = value;
 ```
 
-Example 4: One-way binding (upward, child-to-host) { .caption }
+For example:
 
-```
-<script>
-  Polymer({
-    is: 'custom-element',
-    properties: {
-      someProp: {
-        type: String,
-        notify: true,
-        readOnly: true
-      }
-    }
-  });
-</script>
 
-...
+```html
+<template>
+  <!-- Attribute binding -->
+  <my-element selected$="{{value}}"></my-element>
+  <!-- results in <my-element>.setAttribute('selected', this.value); -->
 
-<!-- changes to "value" are ignored by child due to readOnly:true -->
-<!-- changes to "someProp" propagate upward to "value" on host  -->
-<custom-element some-prop="{{value}}"></custom-element>
+  <!-- Property binding -->
+  <my-element selected="{{value}}"></my-element>
+  <!-- results in <my-element>.selected = this.value; -->
+</template>
 ```
 
-Example 5: Error / non-sensical state { .caption }
 
-```
-<script>
-  Polymer({
-    is: 'custom-element',
-    properties: {
-      someProp: {
-        type: String,
-        notify: true,
-        readOnly: true
-      }
-    }
-  });
-</script>
-...
-<!-- changes to "value" are ignored by child due to readOnly:true -->
-<!-- changes to "someProp" are ignored by host due to square-bracket syntax -->
-<!-- binding serves no purpose -->
-<custom-element some-prop="[[value]]"></custom-element>
-```
+Attribute bindings are always one-way, host-to-target. Values are serialized according to the
```

Comment:
It's host-to-target, so if the host updates the property, the attribute will be added, but if the target element toggles the attribute, the host property is never updated.